src/IssueViz.Security/IssueVizSecurityPackage.cs

```diff
@@ -50,10 +52,27 @@ protected override WindowPane InstantiateToolWindow(Type toolWindowType)
         {
             if (toolWindowType == typeof(HotspotsToolWindow))
             {
+                TestShellLoad();
+
                 return new HotspotsToolWindow();
             }
 
             return base.InstantiateToolWindow(toolWindowType);
         }
+
+        private void TestShellLoad()
```

Comment:
I'll remove this after the PR changes - this is for testing the artifacts in 2015.